Laws/Passed/EA.json

```diff
@@ -0,0 +1,114 @@
+{
+    "Time Proposed": 1605312000,  
+    "Voted Time": null,
+    "Vote ended time": null,
+    "Type": "Law",
+    "Tag": "GOV",
+    "Id": 5,
+    "Repealed": false,
+    "Name": "Electioneering Act",
+    "Short Name": "EA",
+    "Revision": 0, 
+    "Proposer":
+    {
+       "svid":"0",
+       "name":"null"
+    },"Authors": [
+        {
+            "svid":"u-532411ba-fdb2-4ba7-8a7a-c56bbdb8a840",
+            "name":"Justice CSI_Gunner"
```

Comment:
This should really only be their SV name/a name they specify (since we don't want to have to deal with multiple names for a single person)